src/utils/Processing.ts

```diff
@@ -161,6 +162,75 @@ export function readConfig(input: string): Configuration {
   return config;
 }
 
+export async function updateExternalDependencies(config: Configuration): Promise<boolean> {
+  // only try to update if we got the config from sushi-config.yaml, and not from an IG
+  const changedVersions: Map<string, string> = new Map();
+  if (config.filePath != null && config.dependencies?.length > 0) {
+    const promises = config.dependencies.map(async dep => {
+      // current and dev have special meanings, so don't try to update those dependencies
+      if (dep.version != 'current' && dep.version != 'dev') {
+        try {
+          const res = await axiosGet(`https://packages.fhir.org/${dep.packageId}`);
```

Comment:
I'm not sure if this is even something we want to do, but should we also check for packages at `packages2.fhir.org`?

Reply:
At first I was going to say that in theory, `packages2` should only be needed for looking up unreleased versions of FHIR (e.g., FHIR R5), so it's _probably_ OK not to support it here.

But now I'm thinking about it more -- and it's really that `packages` doesn't support R4B or R5 yet.  And I think that means it also doesn't support IGs that depend on R4B or R5.  So `packages2` probably hosts R4B, R5, and all IGs that depend on R4B or R5.  And I think maybe that's a big enough group that we should check both?